util/scripts/install/install.sh

```diff
@@ -12,18 +12,25 @@ if [[ $? -ne 0 ]]; then
   exit 1
 fi
 
+mkdir logs
```

Comment:
If we are going to pipe the output to log files, then we need to add indications that we're doing so and that the process is still going while the output is being piped. It's very confusing behavior, I restarted the install process several times because I thought it was hanging before I realized that everything was being piped to log files.

Reply:
I think its confusing only to us, those who test the install script. I think to ordinary newbies it reduces a lot of overwhelming output.

Reply:
While I do agree that displaying **all** output is too overwhelming for most users, when I'm installing something I expect a certain level of feedback that at least something is happening. Starting the installation and then seeing nothing happening on my screen makes me think nothing is actually happening. I think the solution could be as simple as adding an `echo` statement before each call to an installation section (like `echo "Installing backend dependences..."`) and then adding another when it executes (like `echo "Done!"`). The output would then look something like this:
```
Installing backend dependencies...
Done!
Installing workers...
Done!
Installing nomos...
Done
etc, etc
```
which provides a basic level of feedback that tells people it is in fact working without being overwhelming.